packages/codegen/bin/cli.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
```

Comment:
It might be nice if the main `bin` was actually in `src/bin.ts` and export the `main` function just below, then this "unsafe" / untyped file would be:

```js
#!/usr/bin/env node

import { binMain } from '@data-eden/codegen/bin';

binMain(process.env);
```